src/partials/advantages.html

```diff
@@ -0,0 +1,19 @@
+<div class="section__advantages">
```

Comment:
Section to samodzielna klasa, advantages to druga klasa - blok, brakuje tu kotwicy